application/src/main/java/com/sanction/thunder/email/EmailModule.java

```diff
@@ -39,4 +52,78 @@ EmailService provideEmailService(AmazonSimpleEmailService emailService) {
     return new EmailService(emailService, fromAddress);
   }
 
+  @Singleton
+  @Provides
+  @Named("successHtml")
+  String provideSuccessHtml() {
+    if (successHtmlPath != null) {
+      try {
+        return new String(Files.readAllBytes(Paths.get(successHtmlPath)));
+      } catch (InvalidPathException e) {
+        throw new EmailException("HTML success page path is invalid", e);
+      } catch (IOException e) {
+        throw new EmailException("Error reading file from path: " + successHtmlPath, e);
+      } catch (SecurityException e) {
+        throw new EmailException("Error reading file due to invalid file permissions", e);
+      }
+    }
+
+    try {
+      return Resources.toString(Resources.getResource("success.html"), Charsets.UTF_8);
+    } catch (IOException e) {
+      throw new EmailException("Error reading file from resources folder", e);
+    } catch (IllegalArgumentException e) {
+      throw new EmailException("Default HTML success file was not found in resources folder", e);
+    }
+  }
+
+  @Singleton
+  @Provides
+  @Named("verificationHtml")
+  String provideVerificationHtml() {
+    if (verificationHtmlPath != null) {
+      try {
+        return new String(Files.readAllBytes(Paths.get(verificationHtmlPath)));
+      } catch (InvalidPathException e) {
+        throw new EmailException("HTML verification page path is invalid", e);
+      } catch (IOException e) {
+        throw new EmailException("Error reading file from path: " + successHtmlPath, e);
+      } catch (SecurityException e) {
+        throw new EmailException("Error reading file due to invalid file permissions", e);
+      }
+    }
+
+    try {
+      return Resources.toString(Resources.getResource("verification.html"), Charsets.UTF_8);
+    } catch (IOException e) {
+      throw new EmailException("Error reading file from resources folder", e);
+    } catch (IllegalArgumentException e) {
+      throw new EmailException("Default HTML verification file not found in resources folder", e);
+    }
+  }
+
+  @Singleton
+  @Provides
+  @Named("verificationText")
+  String provideVerificationText() {
+    if (verificationTextPath != null) {
+      try {
+        return new String(Files.readAllBytes(Paths.get(verificationTextPath)));
+      } catch (InvalidPathException e) {
+        throw new EmailException("Text verification page path is invalid", e);
+      } catch (IOException e) {
+        throw new EmailException("Error reading file from path", e);
+      } catch (SecurityException e) {
+        throw new EmailException("Error reading file due to invalid file permissions", e);
+      }
+    }
+
+    try {
+      return Resources.toString(Resources.getResource("verification.txt"), Charsets.UTF_8);
+    } catch (IOException e) {
+      throw new EmailException("Error reading file from resources folder", e);
+    } catch (IllegalArgumentException e) {
+      throw new EmailException("Default Text verification file not found in resources folder", e);
+    }
```

Comment:
Any way we can split some of this code into a `private` function since it's very similar across the three `provides` methods?